src/pages/workspaces/workspace/WorkspaceMenu.test.js

```diff
@@ -279,7 +318,13 @@ describe('WorkspaceMenu - Azure workspace', () => {
     true, false
   ])('renders Share tooltip based on canShare: %s', canShare => {
     // Arrange
-    useWorkspaceDetails.mockReturnValue({ workspace: { canShare, workspace: {} } })
+    useWorkspaceDetails.mockReturnValue({
```

Comment:
Again, test bug-- the workspace details being provided mimic a GCP workspace instead of an Azure one.